src/app/my-profile/components/my-profile-header/my-profile-header.component.ts

```diff
@@ -26,6 +26,7 @@ export class MyProfileHeaderComponent implements OnChanges {
         ? this.reportsService.getPersonMasterDataReports(studentId)
```

Comment:
Wäre es in Zeile 24 nicht ein `switchMap`?

So oder so: Ich habe kein Login gefunden, bei dem dieser Report angezeigt wird.

Reply:
:facepalm: